language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -439,6 +438,8 @@ protected List<LSCompletionItem> getModuleCompletionItems(BallerinaCompletionCon
             completionItems.add(new StaticCompletionItem(ctx, item, StaticCompletionItem.Kind.MODULE));
         });
 
+        //Add prefixes of client declarations
+        completionItems.addAll(this.getClientDeclarationCompletionItems(ctx));
```

Comment:
Extra newline can be removed.
```suggestion
completionItems.addAll(this.getClientDeclarationCompletionItems(ctx));
```